client/src/annoMatrix/views.ts

```diff
@@ -188,10 +237,17 @@ export class AnnoMatrixRowSubsetView extends AnnoMatrixView {
 Utility functions below
 */
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'field' implicitly has an 'any' type.
-function _clipAnnoMatrix(field, colLabel, colSchema, colData, df, qmin, qmax) {
+function _clipAnnoMatrix(
+  field: Field,
+  colLabel: string,
+  colSchema: ColumnSchema,
+  colData: string[],
```

Comment:
colData is not string[], but is any column type supported by Dataframe.  Currently, that is any array (Array<any> or TypedArray).

Return value for this function should be the same type as `colData`

Reply:
Updated views.ts::_clipAnnoMatrix.

Reply:
same issue exists in caller.  I left comments on those functions.  There are types defined in Dataframe that will make this easier.